bind_response.go

```diff
@@ -1,7 +1,25 @@
 package framework
 
+import (
+	"fmt"
+)
+
 // BindResponse represents a response to a BindRequest. It contains a map of credentials and other
 // connection details for a service instance.
 type BindResponse struct {
 	Creds map[string]interface{}
 }
+
+// ResponseCredsToSecretData converts a map[string]interface{}, such as one in BindResponse's
+// Creds field, into a Kubernetes Secret-compatible map[string][]byte
+func ResponseCredsToSecretData(creds map[string]interface{}) map[string][]byte {
```

Comment:
Can we maybe move this functionality into the `k8s/binding` package? Since that's the package that those using the framework (e.g. steward-cf) code to, I'm stuck on the idea of keeping it as minimal as possible.